### ISSUE TITLE
The second argument of `wx.reportAnalytics` should accept object.

### ISSUE THREAD

**Issue by malash** [wechat-miniprogram/api-typings#18]
I guess there is a bug here.

https://github.com/wechat-miniprogram/api-typings/blob/master/types/wx/lib.wx.api.d.ts#L1050-L1056

The second argument of `wx.reportAnalytics` should be `{ [key: string]: any }` instead of `{ key: string, value: any }`.

Document: https://developers.weixin.qq.com/miniprogram/dev/api/wx.reportAnalytics.html

**Comment by wechat-miniprogram-admin** [comment]
it is not a mistake. The second argument of wx.reportAnalytics is an object, which has a key and value parameter.

**Comment by malash** [comment]
I'd like to clarify the issue. The `wx.reportAnalytics` API should accept `{ [key: string]: any }` for these reasons:

1. The demo code in [this doc](https://developers.weixin.qq.com/miniprogram/dev/api/wx.reportAnalytics.html) shows the use case of more than one fields.

```js
wx.reportAnalytics('purchase', {
  price: 120,
  color: 'red'
})
```

2. The config page in WeChat Official Accounts Platform shows that admin can setup more than one fields for customized event.

![image](https://user-images.githubusercontent.com/1812118/56105280-0b2c3a80-5f6e-11e9-87c9-4c32ad94de68.png)

So I believe the typing of second params should be `{ [key: string]: any }` instead of `{ key: string, value: any }`

**Comment by malash** [comment]
@wechat-miniprogram-admin

**Comment by NewFuture** [comment]
Hi @malash,
here is a fork version https://github.com/MS-DG/api-typings
you can help to update this one.

it will push  to @types when stable

**Comment by wechat-miniprogram-admin** [comment]
@malash sorry, i will fix it at 2.7.5 tag
